openadapt/finetune.py

```diff
@@ -0,0 +1,45 @@
+from openadapt import crud, models
+from loguru import logger
+import fire
+
+VALID_MODELS = ("davinci", "mpt-7b")
+
+
+def main(recording_id: int, model_name: str):
+    finetune_recording = crud.get_recording_by_id(recording_id)
+    if not finetune_recording:
+        raise ValueError("A recording with the given id does not exist")
+
+    if model_name.lower() not in VALID_MODELS:
+        raise ValueError("This model is not supported for finetuning yet")
+
+    logger.debug(f"{finetune_recording=}")
+
+
+def davinci_learn_recording_summary_pipeline(
```

Comment:
I think approaching the fine tuning this way (having it learn the intended summary of the recording) is worthwhile because it can then learn some sense of 'meaning' behind each recording, which may produce better results when trying to predict next action events.